test/e2e/mock-e2e.js

```diff
@@ -102,6 +102,9 @@ async function setupMocking(server, testSpecificMock, { chainId }) {
     .forPost(
       'https://arbitrum-mainnet.infura.io/v3/00000000000000000000000000000000',
     )
+    .withJsonBodyIncluding({
+      method: 'eth_chainId',
+    })
```

Comment:
there was a default mock for Arbitrum that another test is using, that returns the chain id to any request made to Arbitrum RPC. We don't want this mock to be applied to all requests, since now we need to mock different requests, so this mock is now only applied when the chainId method is called - preserving the other test functionality, as well as making it work with new Arbitrum mocks added in this PR